better message when people import fake built-ins?

If you `npm add crypto` you get a build error no matter what:

```
> remix build

Building Remix app in production mode...
 > node_modules/@remix-run/node/sessions/fileStorage.js:6:21: error: Could not resolve "crypto" (use "platform: 'node'" when building for node)
    6 │ var crypto = require('crypto');
      ╵                      ~~~~~~~~

 > node_modules/cookie-signature/index.js:5:21: error: Could not resolve "crypto" (use "platform: 'node'" when building for node)
    5 │ var crypto = require('crypto');
```

I'm not sure why our compiler blows up on this (haven't dug in yet) but I know we look at the app deps and ignore built-ins, but maybe we ought to just not allow using any node modules w/ the same name as a built-in node module?

--- a/packages/remix-dev/compiler.ts
+++ b/packages/remix-dev/compiler.ts
@@ -249,7 +249,9 @@ async function createBrowserBuild(
 
   if (fakeBuiltins.length > 0) {
     console.error(
-      `It appears you're using a module that is built in to node, but you installed it as a dependency which could cause problems. Please remove it before continuing.`
+      `It appears you're using a module that is built in to node, but you installed it as a dependency which could cause problems. Please remove ${fakeBuiltins.join(
+        ", "
+      )} before continuing.`
     );
     process.exit(1);
   }

